### PR TITLE
Docs.

### PR DESCRIPTION
**Pull request by pascallouisperez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,31 @@
...
+
+```
+rules:
+  - name: utilities can use other utilities and config
+    packages: util/.*
+    may_depend:
+      - <.*>
+      - third_parties
+      - config
+      - .*util
```
Comment (cbelsole) [inline README.md:16]: Does this also match packages like `github.com/helloeave/eave-go/futil/something`?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,31 @@
...
+    expected:
+      - util/old_and_clunky -> server
+      - old_common
+```
+
```
Comment (cbelsole) [inline README.md]: Can you also provide an example for options?

File: README.md [hunk 3]
```diff
@@ -2,3 +2,31 @@
...
+```
+rules:
+  - name: utilities can use other utilities and config
+    packages: util/.*
+    may_depend:
+      - <.*>
+      - third_parties
+      - config
+      - .*util
+    expected:
```
Comment (cbelsole) [inline README.md]: Maybe deprecated_dependencies?

File: README.md (hunk 1 above)
Comment (pascallouisperez) [reply README.md:16]: Yes, it would

File: README.md (hunk 3 above)
Comment (pascallouisperez) [reply README.md]: Yes, much better name

File: README.md (hunk 2 above)
Comment (pascallouisperez) [reply README.md]: Added, and renamed to `config` instead of `options`

File: README.md (hunk 1 above)
Comment (cbelsole) [reply README.md:16]: I guess it is up to the person who writes the yaml to have good regexes for matching packages.

**Review by cbelsole — APPROVED** [review]

**Pull request merged**